### ISSUE TITLE
Custome Field not removed after Uninstallation

### ISSUE THREAD

**Issue by vishwaravi321** [phamos-eu/it_management#278]
- I uninstalled this app and migrated my site, but after that it shows this error. I mannually removed it from custom form list. But whenever i migrate my site this error shows again and again.
- There's even more custom fields are not removed.
- I have attached the list in task as well. 
- I don't know what other DocTypes this issues are coming:
[![Screenshot_2024-06-24_17-12-41](https://github.com/phamos-eu/it_management/assets/91792810/846b13ac-32e3-49ed-bf73-986cfdc81594)
](url)

![Screenshot_2024-06-24_17-16-12](https://github.com/phamos-eu/it_management/assets/91792810/1c7c6426-cc4b-4df6-91e5-b03ae705ada3)

**Comment by wojosc** [comment]
> I uninstalled this app and migrated my site, but after that it shows this error. I mannually removed it from custom form list. But whenever i migrate my site this error shows again and again.

What did you try and remove?

**Comment by wojosc** [comment]
from what I see, if you remove this field it should work
![image](https://github.com/phamos-eu/it_management/assets/22279621/56cc4cc5-37a8-49b5-a263-14f4e924203e)

**Comment by vishwaravi321** [comment]
> from what I see, if you remove this field it should work ![image](https://private-user-images.githubusercontent.com/22279621/342423270-56cc4cc5-37a8-49b5-a263-14f4e924203e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk1NDY4MDMsIm5iZiI6MTcxOTU0NjUwMywicGF0aCI6Ii8yMjI3OTYyMS8zNDI0MjMyNzAtNTZjYzRjYzUtMzdhOC00OWI1LWEyNjMtMTRmNGU5MjQyMDNlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjI4VDAzNDgyM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWJhZDExMDBlMmQ1YjNkYTBlZWFjMzVlNjEwODM0NjY0MzljZjBhYjQ3MTQyMjE5YzY1OTE4MTk5NzlmZWQ4ZDcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.HkX0v1VLn_gESfVlMiKuuNlFo5goZaySrw3u_E6y3kQ)

**Comment by vishwaravi321** [comment]
Yep that is temporary solution.whenever i migrate the site this fields pop up each time. every time i need to mannually delete this?

**Comment by wojosc** [comment]
thank you for your feedback. We will patch it!

@roquegv please repair this. Thanks.

**Comment by Mathan-V** [comment]
@wojosc I have solved this issue. I can't see or predict your git branching or merging strategy. Can you tell me which branch should i create pull request for. Is it  on develop or straight to v15?

**Comment by wojosc** [comment]
thank you @Mathan-V . Please create your PR towards `develop` and we will then PR it into the the `version-15`

**Comment by Mathan-V** [comment]
@wojosc Thank you for your attention. I have created pull request towards develop branch. Pull Request id #279.
